pkg/demoinfocs/parser.go

```diff
@@ -190,6 +195,11 @@ func (p *parser) Progress() float32 {
 	return float32(p.currentFrame) / float32(p.header.PlaybackFrames)
 }
 
+// IsPOV indicates if the demo being parsed is a POV demo.
+func (p *parser) IsPOV() bool {
```

Comment:
this may lead to some confusion when users call

`NewParser(demoReader).IsPOV()` as it will return false at first and only true after parsing a few frames.

Maybe it's better to only expose this information in the form of an event or something :thinking:

Reply:
👍 yeah I hesitated to add this detail in the comment.
I will replace it with an event similar to `TickRateInfoAvailable`.

Reply:
I replaced it with an event `RecordingPlayerDetected`

Reply:
Can we do `POVRecordingPlayerDetected` - might be more obvious to newcomers

Reply:
Done